lib/components/src/blocks/Story.tsx

```diff
@@ -34,12 +35,16 @@ type IFrameStoryProps = CommonProps;
 
 type StoryProps = InlineStoryProps | IFrameStoryProps;
 
-const InlineStory: FunctionComponent<InlineStoryProps> = ({ storyFn, height, id }) => (
+const InlineStory: FunctionComponent<InlineStoryProps> = ({ storyFn, height, id = uuidv4() }) => (
   <Fragment>
-    {height ? <style>{`#story--${id} { min-height: ${height} }`}</style> : null}
-    <Fragment>
-      {storyFn ? createElement(storyFn) : <EmptyBlock>{MISSING_STORY(id)}</EmptyBlock>}
-    </Fragment>
+    {height ? (
+      <style>{`.sbdocs .story--${id} { min-height: ${height}; transform: translateZ(0); overflow: auto;`}</style>
+    ) : null}
+    <div className={`story--${id}`}>
+      <Fragment>
+        {storyFn ? createElement(storyFn) : <EmptyBlock>{MISSING_STORY(id)}</EmptyBlock>}
+      </Fragment>
+    </div>
```

Comment:
There's no way to do this without the extra `<div>`?

Reply:
I don't see how actually but this Fragment is not necessary anymore if we keep this extra div 🤔

Reply:
@ndelangen Can it fit as it?

Reply:
@ndelangen any concerns?

Reply:
WDYT @shilman, any concerns?